lib/cartopy/examples/nightshade.py

```diff
@@ -0,0 +1,8 @@
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+from cartopy.feature.nightshade import Nightshade
+
+
+ax = plt.axes(projection=ccrs.PlateCarree())
```

Comment:
Need to update the existing example, then.

Reply:
Happy to remove mine. Hadn't actually checked if one existed, my apologies - was using this example as a quick proof-of-concept.

Reply:
It's a part of the Aurora example.